lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -19,7 +20,7 @@ def show_sidebar?
       end
 
       def show_header?
-        step_navs.count == 1
+        step_navs.count == 1 or @active_step_nav_content_id
```

Comment:
I would split this condition out, so that it's clear what's being checked and why. 

e.g. rather than `or @active_step_nav_content_id` maybe use, `or active_step_by_step_journey?`

```
def active_step_by_step_journey?
  active_step_nav_content_id.present?
end
```